soknad/src/main/java/no/nav/k9/søknad/ytelse/pls/v1/PleiepengerLivetsSluttfasePerioderMedEndringUtil.java

```diff
@@ -23,6 +23,7 @@ public static List<Periode> getEndringsperiode(PleipengerLivetsSluttfase ppn) {
     public static List<PerioderMedEndring> getAllePerioderSomMåVæreInnenforSøknadsperiode(PleipengerLivetsSluttfase ppn) {
         var listen = new ArrayList<PerioderMedEndring>();
         listen.add(new PerioderMedEndring().medPerioder("uttak", ppn.getUttak().getPerioder()));
+        listen.add(new PerioderMedEndring().medPerioder("lovbestemtFerie", ppn.getLovbestemtFerie().getPerioder()));
```

Comment:
ppn.getLovbestemtFerie() kan kan/kunne tidligere være null, så det bør nok også være en sjekk mot null eller versjonsnummer, for å unngå NPE

Reply:
Aah godt poeng 👍